src/explore-education-statistics-admin/src/pages/publication/PublicationReleasesPage.tsx

```diff
@@ -19,7 +19,7 @@ const PublicationReleasesPage = () => {
 
       <p>View, edit or amend releases contained within this publication.</p>
 
-      {publication?.permissions.canCreateReleases && (
+      {publication?.permissions?.canCreateReleases && (
```

Comment:
Again, these shouldn't optional - we should be working with `PublicationWithPermissions` from `PublicationContext`